.github/workflows/stale.yml

```diff
@@ -0,0 +1,39 @@
+# This workflow warns of PRs that have had no activity for a specified amount of time.
+#
+# For more information, see https://github.com/actions/stale
+name: Mark stale pull requests
+
+on:
+  # Run every day at 00:00 UTC
+  schedule:
+    - cron: '0 0 * * *'
+  # Or run on demand
+  workflow_dispatch:
+
+jobs:
+  stale:
+
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+
+    steps:
+      - uses: actions/stale@v9
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+
+          days-before-pr-stale: 14
```

Comment:
I wonder if we can should consider 30 days stale time to avoid some noise over holidays etc.

And what if we sent an automated monthly email to dev@ with list of stale PS. That would let both committers and contributors triage and decide whether to close or contribute.

Reply:
Changed to 30 days. 

The monthly email would be a separate bot/script in a different GH action. It would need to use a pre-accepted email address as sender to be allowed to post to dev@ list, and a SMTP server to send from.